src/main/resources/templates/methodNotification.html

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <script>
-		var getOrigin = window.location.origin;
-		window.parent.postMessage("3DS.Notification.Received", getOrigin);
-	</script>
-</head>
-<body></body>
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+
+    <script th:inline="javascript">
+        /*<![CDATA[*/
+
+        var getOrigin = /*[[${origin}]]*/;
```

Comment:
Shouldn't it be

```suggestion
        var getOrigin = [[${origin}]];
```
?

Reply:
CDATA removed